pkg/audit/manager.go

```diff
@@ -905,7 +902,7 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 		if *exportController.ExportEnabled {
 			err := am.exportSystem.Publish(context.Background(), *auditConnection, *auditChannel, violationMsg(constraint, ea, r.ScopedEnforcementActions, gvk, namespace, name, msg, details, labels, timestamp))
 			if err != nil {
-				am.log.Error(err, "error exporting audit violation")
```

Comment:
this should still be logged in the audit pod log

Reply:
This was removed to prevent flood of logs with the same error for each violation in favor of only logging unique errors - https://github.com/open-policy-agent/gatekeeper/pull/3832/files#diff-2a233457fc5c89cf0dd91c044d7582aa1cc0c835feabf4638882ac54ba3ca63aR286.